src/NuGet.Core/NuGet.Packaging.Core/NuspecCoreReaderBase.cs

```diff
@@ -205,6 +213,11 @@ public XDocument Xml
         }
 
         public virtual PackageIdentity GetIdentity()
+        {
+            return GetIdentity(false);
+        }
+
+        public virtual PackageIdentity GetIdentity(bool allowToken)
```

Comment:
where is allowToken being used in this function?

Reply:
Oops, should have been in GetVersion, will fix.